packages/ember-cli-eyeglass/README.md

```diff
@@ -31,12 +32,82 @@ const app = new EmberApp(defaults, {
 });
 ```
 
+Given the following layout:
+
+```sh
+app/styles/app.scss
+           other.scss
+           _shared.scss
+```
+
+
+Will result in:
+
+`assets/<app-name>.css` containing the compiled output of `app/styles/app.scss`
+`assets/foo.css` containing the compiled output of `app/styles/foo.scss`
+
+The contents of `app/styles/_shared.scss` will be include only if required, and will appear in the file which imported them.
+If `app/styles/app.scss` imported `_shared.scss` then `assets/<app-name>.css` will then also contain the contents of `app/styles/_partial.scss`.
+
+
+### Addons
+
+To make an ember-addon (both in-repo and from node_modules) behave as an eyeglass module, add `ember-cli-eyeglass` as a dependency of the addon in question. 
+
+ember-addons have two important directories, `app` and `addon`:
+
+#### my-addon/app/styles
+
+These assets are only considered for top-level addons, nested-addons have their `app` directory ignored.
+
+Given the following folder structure:
+
+```sh
+my-addon/app/styles/
+                   /my-addon.scss
+                   /my-other-file.scss
+                   /-a-partial.scss
+                      
+```
+
+* `my-addon/app/styles` `scss` files will become independent css files in the built output
+* `my-addon/addon/styles` `scss` files will be merged into your applications `vendor.css`;
+
+
+Will result in:
+
+`assets/my-addon.css` containing the compiled output of `my-addon/app/styles/my-addon.scss`
+`assets/my-other-file.css` containing the compiled output of `my-addon/app/styles/my-other-file.scss`
+`-a-partial.scss` will not be included by default, unless the files in `my-app/app/styles/` or `my-addon/app/styles` import it.
```

Comment:
does this mean that a leading dash is an alternate way to indicate files are partials?

Reply:
likely just a typo on my side, i think `_` is the right prefix